<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>222</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tulip Species Predictor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="petal_length_lineedit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="petal_length_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>108</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>31</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Petal Length</string>
    </property>
   </widget>
   <widget class="QLabel" name="petal_width_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>102</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>31</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Petal Width</string>
    </property>
   </widget>
   <widget class="QLabel" name="sepal_length_label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>31</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Sepal Length</string>
    </property>
   </widget>
   <widget class="QLabel" name="sepal_width_label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>105</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>31</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Sepal Width</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="petal_width_lineedit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="sepal_length_lineedit">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="sepal_width_lineedit">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>50</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="prediction_label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>94</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>31</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Prediction:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="predict_button">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>108</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Predict</string>
    </property>
   </widget>
   <widget class="QLabel" name="result_label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>150</y>
      <width>139</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>31</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="accuracy_label">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>190</y>
      <width>45</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>8</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Accuracy:</string>
    </property>
   </widget>
   <widget class="QLabel" name="accuracy_score">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>190</y>
      <width>45</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>8</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
